pages/trending.tsx

```diff
@@ -93,7 +93,7 @@ const Trending = ({ data, query }) => {
 Trending.getInitialProps = async (ctx: NextPageContext) => {
   const trendingQuery = {
     ...ctx.query,
-    ...{ minPopularity: 5, order: 'popularity' as QueryOrder },
+    ...{ minPopularity: 5, minMonthlyDownloads: 250, order: 'popularity' as QueryOrder },
```

Comment:
i'd bump this a bit higher, maybe to 1000. 250/week is very small for a library still imo